realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -182,7 +182,7 @@ public void removeChangeListener(RealmChangeListener listener) {
      * </pre>
      *
      * @return RxJava Observable that only calls {@code onNext}. It will never call {@code onComplete} or {@code OnError}.
-     * @throws UnsupportedOperationException if the required RxJava framework is not on the classpath.
+     * @throws NoClassDefFoundError if the required RxJava framework is not on the classpath.
```

Comment:
It looks a bit odd that the other methods can throw both UnsupportedOperation and NoClassDef, while this only throw the latter?